### PR TITLE
Updating adding profiles

### PR DESCRIPTION
**Pull request by gestchild**

### CODE REVIEW

File: _profiles/article.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
   connectivity: "Cable"
   location: "London_EC2"
   runs: 1
-  url: "https://next.wellcomecollection.org/articles/a-drop-in-the-ocean-daniel-regan"
```
Comment (jamesgorrie) [inline _profiles/article.html:28]: Could we potentially differentiate between WP & Prismic article (making prismic a default, and WP the exception)?

File: _profiles/works.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
   connectivity: "Cable"
   location: "London_EC2"
   runs: 1
-  url: "https://next.wellcomecollection.org/works?query=david&cohort=platformTeam"
```
Comment (jamesgorrie) [inline _profiles/works.html:28]: Do we need the cohort now?

Comment (gestchild) [reply _profiles/works.html:28]: Just using it to test worst case scenario (which is probably more common), where images aren't served by the CDN

Comment (jamesgorrie) [reply _profiles/works.html:28]: Ah, okay, I didn't realise this cohort had this as a feature.

**Review by jamesgorrie — APPROVED** [review]

File: _profiles/article.html (hunk 1 above)
Comment (gestchild) [reply _profiles/article.html:28]: Good idea, will do that in another PR

**Pull request merged**